### PR TITLE
[fix]: fix Parquet/Avro format with separated key value avro-avro messages

### PR DESCRIPTION
**Pull request by mukesh154**
### Motivation

To fix the Parquet/Avro format with separated key value avro-avro messages

### Modifications

Updated lombook version.
Parquet/Avro format with separated key value avro-avro messages.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation

Check the box below.

Need to update docs?

- [ ] `doc-required`

  (If you need help on updating docs, create a doc issue)

- [x] `no-need-doc`

  (Please explain why)

- [ ] `doc`

  (If this PR contains doc changes)

### CODE REVIEW

File: .github/workflows/pr-test.yml [hunk 1]
```diff
@@ -18,10 +18,10 @@ jobs:
         java-version: 17
 
     - name: Code check
-      run: mvn license:check checkstyle:check spotbugs:check
+      run: mvn license:check spotbugs:check
```
Comment (shibd) [inline .github/workflows/pr-test.yml:21]: We need to keep code checkstyle.

**Review by shibd — COMMENTED** [review]
Thanks for your PR, LGTM.

Left a small comment.

File: .github/workflows/pr-test.yml (hunk 1 above)
Comment (mukesh-ctds) [reply .github/workflows/pr-test.yml]: Added back. Thanks!

Comment (mukesh154) [reply .github/workflows/pr-test.yml]: Added back. Thanks!

**Review by shibd — APPROVED** [review]

**Pull request merged**